images/cli-artifacts/Dockerfile.rhel

```diff
@@ -5,11 +5,18 @@ WORKDIR /go/src/github.com/openshift/oc
 COPY . .
 RUN make cross-build --warn-undefined-variables
 
+FROM registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.18-openshift-4.11 AS builder-rhel-7
+WORKDIR /go/src/github.com/openshift/oc
+COPY . .
+RUN make cross-build --warn-undefined-variables
+
 FROM registry.ci.openshift.org/ocp/4.11:cli
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/darwin_amd64/oc /usr/share/openshift/mac/oc
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/darwin_arm64/oc /usr/share/openshift/mac_arm64/oc
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/windows_amd64/oc.exe /usr/share/openshift/windows/oc.exe
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/linux_amd64/oc /usr/share/openshift/linux_amd64/oc
+COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/linux_amd64/oc /usr/share/openshift/linux_amd64/oc.rhel8
```

Comment:
The thought is, as a user, you know what OS version your host is running, but you don't know the OS version of the "default" `/usr/share/openshift/linux_amd64/oc` we ship. 
You could only test it and hope. But, if we have a standard where we always create `oc.rhel?` (for each supported OS), then it becomes predictable and automatable which one you should run on the host.

Reply:
And we are doing this change only in 4.11 because FIPS is always enabled in clusters >= 4.12?

Reply:
We will need to do it for other versions: https://issues.redhat.com/browse/OCPBUGS-23386 . This change is tactical because a customer has hit the issue on 4.11.